recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ogdf" %}
+{% set version = "2022.02" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/refs/tags/dogwood-202202.tar.gz
+  sha256: 308cc2749c6a63520f7979bac86a04066dfea2fb9d3a5e89831318db404bfbf5
+
+  patches:
+    - cmake.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2022.02') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - backward-cpp                           # [build_platform != target_platform]
+    - pugixml                                # [build_platform != target_platform]
+    - coincbc                                # [build_platform != target_platform]
+    - libdwarf-dev                           # [build_platform != target_platform]
+  host:
+    - cmake >=3.1
+    - make               # [not win]
+    - ninja              # [win]
+    - backward-cpp
+    - pugixml
+    - coincbc
+    - libdwarf-dev
```

Comment:
Please address static/underlinking warnings. We do not allow static linking.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=661341&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3106